MapboxNavigation/NavigationMapView.swift

```diff
@@ -709,6 +720,190 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     }
 
     // MARK: - Vanishing route line methods
+
+    var primaryRoutePoints: RoutePoints?
+    var primaryRouteLineGranularDistances: RouteLineGranularDistances?
+    var primaryRouteRemainingDistancesIndex: Int?
+    var routeLineTracksTraversal: Bool = false
+    private var startDate = Date()
+    public var fractionTraveled: Double = 0.0
```

Comment:
Do we need to publicly expose `fractionTraveled`?

Reply:
No, I'll fix it.